admin/views/pupil-status/view-pupil-status.ejs

```diff
@@ -2,37 +2,279 @@
+                <div class="govuk-panel__body custom-card-small-text">
+                    Completed checks
+                </div>
+                <div class="custom-card-large-text">
+                    <%= pupilStatusViewData.pupilsCompletedCount %>
+                </div>
+                <div class="govuk-panel__body custom-card-small-text">
+                    of <%= pupilStatusViewData.totalPupilsCount %> pupils
+                </div>
+            </div>
+            <details class="govuk-details govuk-!-margin-top-5" data-module="govuk-details">
+                <summary class="govuk-details__summary">
+                    <span class="govuk-details__summary-text">
+                      What does each status mean?
+                    </span>
+                </summary>
+                <div class="govuk-details__text">
+                    <dl class="govuk-panel-list">
+                        <dt>
+                            <div><strong>Error in processing</strong></div>
+                            <div>
+                                The pupils check has had an error when being processed.
+                                The pupil will need to take the check again. To do this, assign the pupil a <a href="/restart/select-restart-list">'Restart'</a> and then <a href="/pupil-pin/generate-live-pins-overview">Generate a new PIN</a> for them to take the check again.
+                            </div>
+                        </dt>
+                        <dt>
+                            <div><strong>Pupil check not received</strong></div>
+                            <div>
+                                The pupil has started the check but the time allowed has lapsed without a record of submission. The pupil will need to take the check again. A new PIN can be printed from 8am tomorrow morning.This is the default status when a pupil has been added to the school’s register.
+                                This status will show when:
+                                <ul class="govuk-list govuk-list--bullet">
+                                    <li>A PIN has yet to be generated for pupil to start the check</li>
+                                    <li>A PIN has expired before use</li>
+                                </ul>
+                            </div>
+                        </dt>
+                        <dt>
+                            <div><strong>Not taking the check</strong></div>
+                            <div>
+                                A reason for "not taking the check" has been applied to the pupil.
+                                If removed the pupil returns to the previous state.
+                                To change this go to <a href="/pupils-not-taking-the-check/pupils-list">"Remove pupil from register"</a> page.
+                                The pupils check is being sent to the MTC data base. For technical reasons this is taking longer than expected.
```

Comment:
Is this wording about 'data base' correct?  If the pupil is not taking a check then there isn't a check to send to the database. 😕